src/main/pages/HomePageOnet.java

```diff
@@ -55,30 +52,35 @@ public String acceptCookies2() {
         return getLogoText();
     }
 
-    public void getArticleLinksOpenThemInDifferentTabsAndSaveLinkstoFile() throws IOException {
+    public void getArticleLinksOpenInTabsAndSavetoFile() throws IOException {
         System.out.println("Saving article links to the list");
-        List<WebElement> phraseSearch2 = driver.findElements(By.partialLinkText("ska"));
-        WebElement q;
-        String k;
+        /**
+         Change "ska" to whatever text you want to search it on www.onet.pl site
+         */
+        List<WebElement> searchByPhrase = driver.findElements(By.partialLinkText("ska"));
+        WebElement phraseSearch;
+        String singleArticle;
         List<String> articleLinks = new ArrayList<>();
-        for (WebElement webElement : phraseSearch2) {
-            q = webElement;
-            k = q.getAttribute("href");
-            articleLinks.add(k);
+        for (WebElement webElement : searchByPhrase) {
+            phraseSearch = webElement;
```

Comment:
Redundant, delete

Reply:
You mean here delete lines 64 and 65? I cannot delete this loop, it searches for articles on the page :-D
If you mean delete lines 66 and 67 it's already deleted in this commit.